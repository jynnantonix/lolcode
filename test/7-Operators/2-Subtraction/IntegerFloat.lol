This test makes sure the subtraction operator works correctly on integer and floating point decimal arguments.

HAI
	VISIBLE DIFF OF 2.34 AN 1 MKAY?
KTHXBAI
