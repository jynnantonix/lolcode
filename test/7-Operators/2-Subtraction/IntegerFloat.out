1.3399999999999999 
